models/doctype/Email/Email.js

```diff
@@ -38,6 +38,38 @@ module.exports = {
             "label": "Port",
             "fieldtype": "Int",
             "required": 1
-        }
+        },
+        {
+            "fieldname": "emailAddress",
+            "label": "to",
+            "fieldtype": "Data",
+            "required": 1
+        },
+        {
+            "fieldname": "emailAddress",
+            "label": "cc",
+            "fieldtype": "Data",
+            "required": 0
+        },
+        {
+            "fieldname": "emailAddress",
+            "label": "bcc",
+            "fieldtype": "Data",
+            "required": 0
+        },
+        {
+          "fieldname": "Subject",
+          "label": "Subject",
+          "fieldtype" : "Text",
+          "required": 0
+        },
+        { 
+          "fieldname": "Content",
+          "label": "",
+          "fieldtype" : "Text",
+          "required": 0
```

Comment:
I think this is body, so I'm not sure whether a mail can be send without a body.

Reply:
Blank mails are allowed @meets2tarun . However, for blank subject, we might have to add a warning/prompt later "Send mail without any subject", but maybe we'll deal with that while adding interface